js/index.js

```diff
@@ -25,4 +25,47 @@ for (let skill of skills) {
   let newskill = document.createElement("li");
   newskill.innerHTML = skill;
   skillsList.appendChild(newskill);
-}
\ No newline at end of file
+}
+
+const messageForm = document.querySelector("[name='leave_message']");
+messageForm.addEventListener("submit", handleSubmit);
+
+function handleSubmit(e){
+  e.preventDefault();
+  const name = e.target.usersName.value;
+  const email = e.target.usersEmail.value;
+  const message = e.target.usersMessage.value;
+  console.log(name, email, message);
+  
+  //Messages in List
+  const messageSection = document.querySelector('#messages');
+  const messageList = messageSection.querySelector("ul");
+  const newMessage = document.createElement("li");
+  
+  newMessage.innerHTML = `<a href="mailto:${usersEmail}">${usersName}</a>\n
```

Comment:
these need to reference variables created on lines 35-37

`usersEmail`, `usersName` are don't seem like defined variables within this js file